### PR TITLE
Feat: assume UTF-8 encoding for commit messages

### PR DESCRIPTION
**Pull request by thekaveman**
Follow up to #49, see comment thread for details.

### CODE REVIEW

**Comment by thekaveman** [comment]
@Nagico mind taking a look at this PR and see if it works for you? 

I built on top of your PR (and helpful tests!), just slightly modified to hardcode the encoding.

File: README.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ repos:
       - id: conventional-pre-commit
         stages: [commit-msg]
         args: [] # optional: list of Conventional Commits types to allow e.g. [feat, fix, ci, chore, test]
+                 # if you want to set encoding, use --encoding=<encoding>, before types
```
Comment (Nagico) [inline README.md:29]: unnecessary line

File: README.md [hunk 2]
```diff
@@ -93,12 +94,12 @@ Then run the command line script:
...
 
-Where `[types]` is an optional list of Conventional Commit types to allow (e.g. `feat fix chore`)
+- `[types]` is an optional list of Conventional Commit types to allow (e.g. `feat fix chore`)
 
-And `input` is a file containing the commit message to check:
+- `input` is a file containing the commit message to check:
 
 ```shell
-conventional-pre-commit feat fix chore ci test .git/COMMIT_MSG
+conventional-pre-commit --encoding=utf-8 feat fix chore ci test .git/COMMIT_MSG
```
Comment (Nagico) [inline README.md:102]: unnecessary `--encoding=utf-8`

**Comment by Nagico** [comment]
Thanks a lot! I try it on the Pycharm, VSCode and terminal, and all of them work well. 
There are some unnecessary changes in the readme file. You can simply delete them.

**Pull request merged**